services/webdav/pkg/service/v0/service.go

```diff
@@ -242,7 +243,7 @@ func (g Webdav) SpacesThumbnail(w http.ResponseWriter, r *http.Request) {
 		default:
 			renderError(w, r, errInternalError(err.Error()))
 		}
-		g.log.Error().Err(err).Msg("could not get thumbnail")
+		logger.Error().Err(err).Msg("could not get thumbnail")
```

Comment:
@C0rby @butonic I think this could be level warn. Could the admin do anything about it when we land in this codepath?

Reply:
debug